code/index.html

```diff
@@ -1,30 +1,135 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=New+Rocker|Roboto&display=swap" rel="stylesheet">
+  <title>Drummer Bootcamp</title>
+
+</head>
+
+<body>
+
+  <header>
+    <div class="video-container">
+      <video autoplay muted loop playsinline>
+        <source class="video-mobile" src="./videos/drummerVideo360.mp4" type="video/mp4"
+          alt="by Taryn Elliott from Pexels">
+        <source class="video-tablet" src="./videos/drummerVideo960.mp4" type="video/mp4"
+          alt="by Taryn Elliott from Pexels">
+        <source class="video-desktop" src="./videos/videoDrummer-1280.mp4" type="video/mp4"
+          alt="by Taryn Elliott from Pexels">
+        Your browser does not support the video tag.
+      </video>
+      <div class="overlay-header">
+        <h1 class="h1-header">Drummer Bootcamp</h1>
+        <h2 class="h2-header">Become an animal with our help!</h2>
```

Comment:
How come you need these kind of classes to your headings? In comparence with just styling `h1` in the css? I'm just curious!